server/bin/state/config/pbench-server.cfg

```diff
@@ -49,7 +49,7 @@ tasks = pbench-sync
 satellite-host = pbench-satellite.example.com
 satellite-prefix = ONE
 satellite-lock = pbench-sync-satellite-%(satellite-prefix)s.lock
-satellite-archive = %(unittest-dir)s/pbench-satellite/archive/fs-version-001
+satellite-archive = %(unittest-dir)s/pbench-satellite/archive/fs-version-002
```

Comment:
This should not be changed. The archive fs-version is unrelated to the reception area fs-version.